db/professions/enchanting/skills.lua

```diff
@@ -126,7 +126,7 @@ TRADE_SKILLS_DATA_TURTLE["skills"]["Enchanting"] =
             },
         },
         {
-            ["id"] = 1708151578,
+            ["id"] = 45071,

```

Comment:
How do you find this ID?

https://database.turtle-wow.org/?item=51286 there is no link to spell here 
https://database.turtle-wow.org/?spell=45071 and there is no valid spell here

Reply:
I noticed the ID showing up when scanning through my chars known skills but not being in the available table. Superwow's `SpellInfo()` function returns `Enchant 2H Weapon - Minor Intellect` as the name for that spell ID.